src/Management/test/Endpoint.Test/ActuatorsHostBuilderTest.cs

```diff
@@ -194,7 +196,7 @@ public async Task HealthActuatorWithoutDefaultContributors(HostBuilderType hostB
     {
         var appSettings = new Dictionary<string, string?>(AppSettings)
         {
-            { "Management:Endpoints:Health:ShowDetails", "Always" },
+            { "Management:Endpoints:Health:ShowComponents", "Always" },
```

Comment:
These settings don't match the purpose of this test anymore, looking at its name: HealthActuatorWithoutDefaultContributors. It should only set:
```
            { "Management:Endpoints:Health:DiskSpace:Enabled", "false" },
            { "Management:Endpoints:Health:Ping:Enabled", "false" },
```